docs/userGuide/syntax/navBars.mbdf

```diff
@@ -69,13 +73,42 @@ If you wish to further customize your navbar beyond the primary, dark, and light
 <md>In `{your-site}/_markbind/headers/header.md`, you can change `<navbar type="dark/primary/light">` to `<navbar type="none" add-class="bg-warning/danger/info/primary/success">` to apply Bootstrap background styles.</md>
 </modal>
 
+**Navbars let you specify rules to customise link highlighting.**
+- Define `default-highlight-on` in `<navbar>` to specify fallback highlight rules.
+- Define `data-highlight` in `<a>` tags with the class `nav-link` or `dropdown-item` to specify individual highlight rules.
+
+<include src="codeAndOutput.md" boilerplate >
```

Comment:
since its a little hard to demonstrate this on one page, how about `<include>`ing the userGuide navbar? (just the code snippet) :-)